astro/terraform/terraform_remote_state_disable_utils.go

```diff
@@ -107,7 +137,7 @@ func deleteTerraformBackendConfigFromFile(file string) error {
 	return nil
 }
 
-func parseTerraformConfig(in []byte) (*ast.ObjectList, error) {
+func parseTerraformConfigWithHCL(in []byte) (*ast.ObjectList, error) {
```

Comment:
Does this function apply only to HCL 1.0?

If so, I'd name it: `parseTerraformConfigWithHCL1`.